Install-DVWA.sh

```diff
@@ -31,18 +31,34 @@ check_program() {
     fi
 }
 
-# MySQL root password prompt function
-get_mysql_root_password() {
-    read -s -p "$(get_language_message "Enter MySQL root password (press Enter for no password): " "Ingrese la contraseña de root de MySQL (presione Enter para ninguna contraseña): ")" mysql_root_password
-
-    # Verificar si la contraseña está vacía
-    if [ -z "$mysql_root_password" ]; then
-        echo -e "\n$(get_language_message "No password provided." "No se proporcionó contraseña.")"
+run_mysql_commands() {
+    local mysql_user
+    local mysql_password
+
+    read -p "$(get_language_message "\e[96mEnter MySQL user:\e[0m " "\e[96mIngrese el usuario de MySQL:\e[0m ")" mysql_user
```

Comment:
We need some more prompting here. A lot of people using this script will be doing it with no knowledge of what is going on, asking them for a password that they don't know will confuse them.

Tell them the default creds on a fresh install are root / <empty>

Reply:
I added something like this should be enough

echo -e "\n$(get_language_message "\e[96mDefault credentials:.\e[0m" "\e[96mCredenciales por defecto:\e[0m")"
    echo -e "Username: \033[93mroot\033[0m"
    echo -e "\n$(get_language_message "Password: \033[93m[No password just hit Enter]\033[0m" "Password: \033[93m[Sin contraseña solo presiona Enter.]\033[0m")"